### DISCUSSION TITLE
Access denied

### DISCUSSION THREAD

**Discussion by Kokepdl** [GregoryAM-SP/The-Minecraft-Overviewer#77] [Q&A]
I followed all the steps in the guide but when it came time to run the .bat it denied me access and then proceeded to close the cmd. I already tried opening it with administrator and restarting the PC but no difference, I don't know if it's my fault or the fault of the pc
![Captura de pantalla 2024-09-25 023331](https://github.com/user-attachments/assets/6e171dd6-c3fb-44ec-8a96-f75371fe8fa2)

**Comment by stwalkerster** [comment]
I've not been able to reproduce this issue as reported.

## What I did:
1. Downloaded a fresh copy of the latest Windows build from https://overviewer.gregoryam.com/download/
2. Extracted the zip file using Windows Explorer to `C:\Users\simon\Downloads\overviewer\`
3. Copied my test world onto the system to `C:\Users\simon\Desktop\render_test\`
4. Downloaded the latest Minecraft client jar from https://overviewer.org/textures/1.21 to `C:\Users\simon\Downloads\client.jar`
5. Created a new folder at `C:\Users\simon\Downloads\ovconfig`
6. Created a new file at`C:\Users\simon\Downloads\ovconfig\config.py` containing the below configuration
7. Opened a new command prompt at `C:\Users\simon\Downloads\overviewer\` and ran the command `overviewer.exe --config="C:\Users\simon\Downloads\ovconfig\config.py" 1>>C:\Users\simon\Downloads\ovconfig\log.txt`

The command completed successfully, and generated a map output at `C:\Users\simon\Desktop\output\`

### Configuration
```python
worlds["My world"] = "C:/Users/simon/Desktop/render_test"

texturepath = "C:/Users/simon/Downloads/client.jar"

renders["normalrender"] = {
    "world": "My world",
    "title": "Normal Render of My World",
}

outputdir = "C:/Users/simon/Desktop/output"
```

## Thoughts
* Is the file at `C:\Users\koked\Documents\SampleOverviewerFiles\Log` writable? Check the read only flag and the security permissions on the `Log` file and the containing `SampleOverviewerFiles` folder
* Is `C:\Users\koked\Documents\SampleOverviewerFiles\Log` actually a file, or is it a folder? It need not exist (a file will be created there), but it cannot be a folder.
* Try running Overviewer without the redirection (from the `1>>` to the end).
* Check the security permissions on `overviewer.exe` includes the "Read and execute" flag
* Check any security software you have running that might be getting in the way.

**Comment by Kokepdl** [comment]
Follow the steps you told me but when I run the command it gives me this

![Captura de pantalla 2024-09-25 184005](https://github.com/user-attachments/assets/00c7d7e0-4d0e-4fb9-9f87-9539ee769da6)

The log file you told me about was a folder, and I already deleted it and tried the sample again and it gives me the same error in config.py and deleted 1>> the command does not run for some reason, I also deactivated windows defender in case that was the problem and it gives me the same error

**Comment by stwalkerster** [comment]
This is clearly an improvement on your last situation as Overviewer has made progress.

I think your problem now is your use of backslashes (`/`) in the file paths. Python uses backslashes as an escape character, so you should either double up the backslashes or replace with forward slashes (`/`). See my example config file above.

**Comment by Kokepdl** [comment]
I change the backslashes in the python and now de cmd doesn't do anything, not even a error and where do i put the double backslashes

**Comment by stwalkerster** [comment]
Please can you provide a copy of your configuration file?
